lib/octokit/client/deployments.rb

```diff
@@ -6,6 +6,17 @@ class Client
     # @see https://developer.github.com/v3/repos/commits/deployments/
     module Deployments
 
+      # List a single deployment for a repository
+      #
+      # @param repo [Integer, String, Repository, Hash] A GitHub repository
+      # @param deployment_id [Integer, String, Repository, Hash] A GitHub repository
+      # @return <Sawyer::Resource> A single deployment
+      # @see https://developer.github.com/v3/repos/deployments/#get-a-single-deployment
+      def deployment(repo, deployment_id, options = {})
+        get("#{Repository.path repo}/deployments/#{deployment_id}", options)
+      end
+      alias :list_deployment :deployment
```

Comment:
This alias doesn't really make sense, could we remove it?

Reply:
I could but I was following the pattern of the other methods. Does list mean "array" in this case or "look".

Reply:
In this case list means "array"

Reply:
Cool! I'll fix